mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -17,6 +17,8 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 			COMMENT the entity to choose equipments from
 		ARG 2 condition
 			COMMENT extra conditions for the item stack to pass for selection
+	METHOD method_24366 (Lnet/minecraft/class_1799;)Z
```

Comment:
Mind map this method?

Reply:
It's the lambda in `chooseEquipment`, so the method cannot be mapped, only its lambda param can be